server/modules/record/repository/nodeRepository.js

```diff
@@ -7,6 +7,8 @@ const DbUtils = require('@server/db/dbUtils')
 const Node = require('@core/record/node')
 const { getSurveyDBSchema, disableSurveySchemaTableTriggers, enableSurveySchemaTableTriggers } = require('../../survey/repository/surveySchemaRepositoryUtils')
 
+const nodeTableColumns = ['uuid', 'date_created', 'date_modified', 'record_uuid', 'parent_uuid', 'node_def_uuid', 'value', 'meta'] //used for node values batch insert
```

Comment:
this can just be called ```tableColumns```, what do you think?

Reply:
right